packages/vm/src/identity_digest.rs

```diff
@@ -0,0 +1,45 @@
+//! Dummy 256-bits Digest impl.
+//! This digest stores/accepts a value of the proper length.
+//! To be used for / with already hashed values, just to comply with the Digest contract.
+//!
+//! Adapted from `sha2` [sha256.rs](https://github.com/RustCrypto/hashes/blob/master/sha2/src/sha256.rs)
+use sha2::digest::consts::U32;
+use sha2::digest::generic_array::GenericArray;
+use sha2::digest::{FixedOutputDirty, Reset, Update};
+
+use sha2::digest;
```

Comment:
We can import all of those from the `digest` crate directly. No need to have sha2 in here.

https://docs.rs/digest/0.9.0/digest/index.html